ui/js/reducers/claims.js

```diff
@@ -78,6 +81,7 @@ reducers[types.FETCH_CLAIM_LIST_MINE_COMPLETED] = function(state, action) {
   return Object.assign({}, state, {
     isFetchingClaimListMine: false,
     myClaims: myClaims,
+    allMyClaimsByTxidNout: allMyClaimsByTxidNout,
```

Comment:
This set doesn't need to be stored in state. It can be generated from `myClaims` via a selector.

Reply:
I think `myClaims` only filter out claims which are strictly claims(not updates, supports and tips), so I had to make this as a separate state.

Reply:
So then perhaps the filtering should happen in the other direction, but it still seems like we're storing redundant data and this could be avoided.

Maybe it'd make sense that `myClaims` include _all_ of your claims, and instead the relevant `selectXXX` calls in `selectors/claims.js` be updated and renamed as appropriate.

Reply:
I agree. I'd have to look deeper into it, as there is code dependent on this.